blivet/devices/md.py

```diff
@@ -534,6 +536,30 @@ def _post_create(self):
         for member in self.members:
             member.format.md_uuid = self.uuid
 
+        def remove_stale_lvm():
+            """ Remove any stale LVM metadata that pre-existed in a new array's on-disk footprint. """
+            log.debug("waiting 5s for activation of stale lvm on new md array %s", self.path)
+            time.sleep(5)
+            udev.settle()
+            pvs_info.drop_cache()
+            pv_info = pvs_info.cache.get(self.path)
+            if pv_info is None:
+                return
+
+            if pv_info.vg_uuid:
+                log.info("removing stale LVM metadata found on %s", self.name)
+                try:
+                    blockdev.lvm.vgremove(pv_info.vg_uuid)
```

Comment:
So is this somehow working when the CLI-based LVM plugin is used by *libblockdev*?

Reply:
No, I apparently pushed the wrong code.